src/index.js

```diff
@@ -2,13 +2,30 @@ const t = require('@babel/types');
 const extractArguments = require('./extractArguments');
 const combineArguments = require('./combineArguments');
 
+const LIBS = ['clsx', 'classnames'];
+
 module.exports = () => {
+  let names = [];
   return {
     visitor: {
+      Program: {
+        enter() {
+          names = [];
+        },
+      },
+      ImportDeclaration(path) {
```

Comment:
Do you want to extend this to handle `const cx = require('classnames')`?

Reply:
Great idea. It's a little odd to use `require` nowadays in the client side code but it is something that should be handled too. Thanks